README.md

```diff
@@ -17,13 +17,26 @@ An incubating AI-powered Q&A for your codebase.
 Make sure [git-lfs](https://git-lfs.com/) is installed.
 
 ```bash
-# Clone the repository
+# 1. Clone the repository
 git clone https://github.com/TabbyML/quick-question.git
 
-# Save your OPENAI_API_KEY to file as secret
+# 2. Save your OPENAI_API_KEY to file as secret
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
-# Start container
+# 3. Run Quick Question on your github project.
+# 3.1. Create a directory in './data' for your project, replace 
+#      'your_project_name' with your own.
+mkdir ./data/your_project_name
+
+# 3.2. Copy metadata template to your project directory.
+cp ./data/metadata_example.json ./data/your_project_name/metadata.json
```

Comment:
Maybe embed the example content here directly. No need to checkin a separate metadata_example.json file.